indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -70,7 +75,12 @@
             self.element.addClass('categorynav');
             self._createList();
             self._createSearchField();
-            self.goToCategory(self.options.categoryId);
+            self._createBindings();
+            if (typeof self.options.category === 'object') {
```

Comment:
I think there's `_.isObject()` - AFAIK *typeof object* checks are not such a good idea in most cases: http://stackoverflow.com/questions/3787901/why-javascripts-typeof-always-return-object